libs/developers-api-client/src/main/kotlin/com/linecorp/link/developers/client/response/responses.kt

```diff
@@ -268,6 +268,12 @@ data class NonFungibleTokenTypeHolder(
     val numberOfIndex: String
 )
 
+data class NonFungibleTokenTypeHolderList(
+    val list: List<NonFungibleTokenTypeHolder>,
+    val prePageToken: String? = null,
+    val nextPageToken: String? = null,
```

Comment:
Based on the test case, I think this is more appropriately of type `String` than `String?`.